src/Mod/Sketcher/App/GeometryFacade.cpp

```diff
@@ -67,17 +68,17 @@ std::unique_ptr<GeometryFacade> GeometryFacade::getFacade(Part::Geometry* geomet
     // return std::make_unique<GeometryFacade>(geometry);
 }
 
-std::unique_ptr<const GeometryFacade> GeometryFacade::getFacade(const Part::Geometry* geometry)
-{
-    if (geometry) {
-        return std::unique_ptr<const GeometryFacade>(new GeometryFacade(geometry));
-    }
-    else {
-        return std::unique_ptr<const GeometryFacade>(nullptr);
-    }
-    // make_unique has no access to private constructor
-    // return std::make_unique<const GeometryFacade>(geometry);
-}
+// std::unique_ptr<const GeometryFacade> GeometryFacade::getFacade(const Part::Geometry* geometry)
+// {
+//     if (geometry) {
+//         return std::unique_ptr<const GeometryFacade>(new GeometryFacade(geometry));
+//     }
+//     else {
+//         return std::unique_ptr<const GeometryFacade>(nullptr);
+//     }
+//     // make_unique has no access to private constructor
+//     // return std::make_unique<const GeometryFacade>(geometry);
+// }

```

Comment:
If we're leaving this code here there should be an explanation of why.

Reply:
Does literally the same thing as another override. Probably safe to delete (for now).